.travis.yml

```diff
@@ -7,6 +7,8 @@ scala:
 - 2.11.12
 - 2.12.4
 
+sbt_args: -J-Xss4M
```

Comment:
I don't think this is doing anything.  The build ran with `Xss6M` by the look of it:

https://travis-ci.org/underscoreio/slickless/jobs/357367912